### ISSUE TITLE
Stopped working in newest version of Plex Media Server

### ISSUE THREAD

**Issue by RoscoeD** [ukdtom/ExportTools.bundle#74]
I was forced to upgrade my Server because my Player auto-updated the Plex app...and perhaps a coincidence (or perhaps not) ExportTools stopped working.  It says an internal error has occurred.  Most recent export tools log is attached as well as the entire log download.

I've become quite dependent on this tool and now am stressed over its not working.  Any help would be greatly appreciated.

Thanks

[Plex Media Server Logs_2018-12-09_00-25-22.zip](https://github.com/ukdtom/ExportTools.bundle/files/2660367/Plex.Media.Server.Logs_2018-12-09_00-25-22.zip)

[com.plexapp.plugins.ExportTools.log](https://github.com/ukdtom/ExportTools.bundle/files/2660365/com.plexapp.plugins.ExportTools.log)

**Comment by ukdtom** [comment]
According to the logs, the user that runs Plex has no access to where you want it to put the output file

`OSError: [Errno 13] Permission denied: '/Volumes/MKV Movies'`

**Comment by RoscoeD** [comment]
Very strange...I spent an hour this am trying to regain write permissions and nothing would work.  Tried running "First Aid" on the drive (Mac OS) and it couldn't unmount the drive to evaluate what was wrong.  So in Plex I removed the drive and re-added it so it wouldn't be the first drive in the list.  The files still ended up there, but it worked.

Still don't have permission editing ability, but now it appears I don't need them.

Thanks!
